ldms/src/sampler/zfs_leafvdevs/zfs_leafvdevs.c

```diff
@@ -0,0 +1,496 @@
+/* -*- c-basic-offset: 8 -*- */
+/* Copyright 2022 Lawrence Livermore National Security, LLC
+ * See the top-level COPYING file for details.
+ *
+ * SPDX-License-Identifier: (GPL-2.0 OR BSD-3-Clause)
+ */
+
+#define _GNU_SOURCE
+#include <stdlib.h>
+#include <ctype.h>
+#include <glob.h>
+#include <string.h>
+#include <getopt.h>
+#include <stdio.h>
+#include <stdint.h>
+#include <inttypes.h>
+#include <sys/types.h>
+#include <unistd.h>
+#include <libzfs.h>
+#include <libzutil.h>
+#include <stdlib.h>
+#include <string.h>
+#include <unistd.h>
+#include "ldms.h"
+#include "ldmsd.h"
+#include "config.h"
+#include "sampler_base.h"
+#include "libzfs.h"
+#include <stddef.h>
+
+#define SAMP "zfs_leafvdevs"
+#ifndef PATH_MAX
+#define PATH_MAX 2048
+#endif
```

Comment:
`PATH_MAX` is unused (and 4096 by default on Linux).

Reply:
MAXI_PATH is gone.